<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="10">
      <widget class="QLabel" name="CurrentNote">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;C0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLCDNumber" name="SustainLCD">
       <property name="value" stdset="0">
        <double>-20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QDial" name="SustainSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="9" alignment="Qt::AlignRight">
      <widget class="QSlider" name="NoteSelector">
       <property name="maximum">
        <number>107</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Attack s</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="CurrentFrequency"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <widget class="QDial" name="AttackSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="10" alignment="Qt::AlignHCenter">
      <widget class="QSlider" name="FreeAmplitude">
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLCDNumber" name="ReleaseLCD">
       <property name="value" stdset="0">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sustain value</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Decay s</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Release s</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLCDNumber" name="AttackLCD">
       <property name="value" stdset="0">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QDial" name="ReleaseSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QDial" name="DecaySlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLCDNumber" name="DecayLCD">
       <property name="value" stdset="0">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QDial" name="VolumeControl">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="6">
      <widget class="QComboBox" name="SetSource">
       <item>
        <property name="text">
         <string>Sinus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rect</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sawtooth</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Play Automatically</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="VolumeLCD">
       <property name="value" stdset="0">
        <double>-50.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
